### DISCUSSION TITLE
Intal AX200 & RTL 8822BE - Test Results

### DISCUSSION THREAD

**Discussion by ThatzOkay** [vanilla-wiiu/vanilla#18] [Hardware Results]
Tested this card ony my old lenovo laptop. Seemd to work fine. Except no microphone when looking in the system settings then tablet then microphone test. The RTL 8822BE also works just no microphone again

Tested on arch linux with latest kernel and packages

**Comment by itsmattkc** [comment]
Thanks for testing! I've added your results to the wiki. Don't worry about the microphone not working, I haveen't implemented it yet, but it'll be in soon.

**Comment by Mickmick21** [comment]
Can confirm it also works on a Surface Go 2 with no issues, as it also uses an Intel AX200.

**Comment by SirRGB** [comment]
Same for the Tuxedo Pulse 15 Gen 2 (SCHENKER VIA 15 Pro rebrand likely), uses the Intel AX200 as well

**Comment by Nadhorion** [comment]
Dell Inspiron 5515, Intel AX200, Linux mint 22.3. 

It connected to my console for about 20 seconds with random lag and then disconnected. the screen said it was reconnecting and after a minute I tried to resync my console again but it didn't seem like it would connect anymore.
